packages/cozy-intent/src/api/services/NativeService.ts

```diff
@@ -54,23 +65,27 @@ export class NativeService {
   ): Promise<Connection> => await ParentHandshake(messenger, this.localMethods)
 
   public tryEmit = async (event: NativeEvent): Promise<void> => {
-    if (!StaticService.isPostMeMessage(event)) return
+    if (!this.isPostMeMessage(event)) return
```

Comment:
should we insert in the return type it can return void directly?

Reply:
TS will refuse that since the method is declared as async